tests/Unit/SprykerEco/Zed/ArvatoRss/Business/Api/Mapper/RiskCheckResponseMapperTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Unit\SprykerEco\Zed\ArvatoRss\Business\Api\Mapper;
+
+use ArrayObject;
+use Generated\Shared\Transfer\ArvatoRssRiskCheckResponseTransfer;
+
+class RiskCheckResponseMapperTest extends AbstractMapperTest
+{
+
+    /**
+     * @dataProvider responseDataProvider
+     *
+     * @param \ArrayObject $data
+     *
+     * @return void
+     */
+    public function testMapResponseToQuote($data)
+    {
+        $mapper = $this->helper->createResponseMapper();
+
+        $expected = $this->createExpectedTransfer($data)->toArray(true);
+        $actual = $mapper->mapResponseToQuote(
+            $this->createResponseTransfer($data),
+            $this->quoteHelper->createQuoteTransfer()
+        )->toArray(true);
+
+        $this->assertEquals($expected, $actual);
+    }
+
+    /**
+     * @return array
+     */
+    public function responseDataProvider()
+    {
+        return [
+            [
+                new ArrayObject(
+                    [
+                        'result' => 'R',
```

Comment:
Are you sure these keys are not restricted ?
I think those should be used from one of consts files